pages/search/_keyword.vue

```diff
@@ -153,14 +160,35 @@ export default class SearchPage extends Vue {
     return !!this.errorMessage
   }
 
-  async mounted() {
+  mounted() {
+    this.search()
+  }
+
+  @Watch('$route.query')
+  onQueryChange() {
+    this.search()
+  }
+
+  async search(){
     logTrackerEvent(this, 'ISCNSearch', 'ISCNSearchResult', this.queryAllTerm, 1)
     if (this.queryAllTerm)  {
       await this.queryISCNByKeyword(this.queryAllTerm)
     } else {
-      const { keyword, keywords } = this.$route.query;
+      const { 
+        iscnId,
+        owner,
+        contentFingerprint,
+        stakeholderId,
+        stakeholderName, 
+        keyword,
+      } = this.$route.query;
       const searchObject = {
-        keyword: (keyword || keywords) as string,
+        iscnId: iscnId as string,
+        owner: owner as string,
+        contentFingerprint: contentFingerprint as string,
+        stakeholderId: stakeholderId as string,
+        stakeholderName: stakeholderName as string,
+        keyword: keyword as string,
```

Comment:
To simplify the search object, I keep the `keyword` and delete the `keywords`.
For `stakeholderId`, `stakeholderName`, and `keyword`, it's possible to declare as `string[]` to set multiple conditions.
But since the use case is clicks and search, I use the single term for now.
Is it preferred to implement with plural?

Reply:
I think single item is good enough for now